lib/hydra/derivatives.rb

```diff
@@ -118,7 +118,7 @@ def constantize_processor(processor)
     end
 
     def transform_datastream(file_name, transform_directives, opts={})
-      transform_file(file_name, transform_directives, opts={})
+      transform_file(file_name, transform_directives, opts)
```

Comment:
This looks exactly like what @stkenny was asking for in #58.

This will not alter the default behavior of an empty options hash leads to an image processor.